CONTRIBUTING.md

```
@@ -0,0 +1,368 @@
+- Include icons from `lucide-react` or `sanity/icons`
+- Use German titles and descriptions for content editors
+- Follow the established schema folder structure
+
+**Example:**
+
+```ts
+import { FileText } from 'lucide-react';
+import { defineField, defineType } from 'sanity';
+
+export const mySchemaType = defineType({
+	fields: [
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)